roles/openshift_preflight/common/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+# check content available on all hosts
+- when:
+    - not containerized
+  block:
+
+    - name: determine if yum update will work
+      action: check_yum_update
+      register: r
+
+    - set_fact:
+        oo_preflight_check_results: "{{ oo_preflight_check_results + [r|combine({'_task': 'check content available on all hosts'})] }}"
```

Comment:
> Variable overriding is a thing, and even the precedence order is listed in the docs (we only use set_fact).

Yes, variable overriding is supported and predictable. The thing that I'm not sure is officially supported and reliable is referencing the overridden variable when overriding the variable.